src/components/IIIFThumbnail.js

```diff
@@ -11,12 +11,12 @@ import getThumbnail from '../lib/ThumbnailFactory';
  */
 export class IIIFThumbnail extends Component {
   /** */
-  static getUseableLabel(resource, index) {
+  static getUseableLabel(resource, langs) {
     return (resource
       && resource.getLabel
       && resource.getLabel().length > 0)
-      ? resource.getLabel().getValue()
-      : String(index + 1);
+      ? resource.getLabel().getValue(langs)
+      : resource.id;
```

Comment:
Why `resource.id` instead of the index value?

Reply:
Sorry about that, it was probably an ill-advised attempt a a unification of the various `getUsableLabel` methods that went nowhere in the end, I'll revert!